Taskfile.yml

```diff
@@ -172,13 +163,32 @@ tasks:
       - if [ "{{.ARCHIVETYPE}}" = ".zip" ]; then zip -j -r {{.ARCHIVE}} {{.EXECUTABLE}}; fi
       - if [ "{{.ARCHIVETYPE}}" = ".gz" ]; then gzip -v -c {{.EXECUTABLE}} > {{.ARCHIVE}} ; fi
     vars:
+      GOOS: '{{default OS .GOOS}}'
```

Comment:
minor: Why reorder GOOS to the top?

Feels like it should be next to GOARCH?

Reply:
VARs are evaluated sequentially - so we need it set first in case OS is used to override the value. Current usage doesn't run in to the problem, but fixing just in case.